craft_cli/messages.py

```diff
@@ -89,6 +99,68 @@ def _get_log_filepath(appname: str) -> pathlib.Path:
     return basedir / filename
 
 
+class _Spinner(threading.Thread):
+    """A supervisor thread that will repeat long-standing messages with a spinner besides it."""
+
+    def __init__(self, printer: "_Printer"):
+        super().__init__()
+        # special flag used to stop itself
```

Comment:
Maybe better as `special flag used to stop the spinner thread`? (otherwise it seems that itself == the flag)

Reply:
ah, good idea, thanks!